src.ts/utils/format.ts

```diff
@@ -0,0 +1,37 @@
+const BN = require ('bn.js');
+
+// Exponent for calculating how many units of account balance are in one near.
+const exp = 24;
```

Comment:
nit: constants ideally should be ALL_CAPS.  So makes sense to use something like `NEAR_NOMINATION_EXP`.

Reply:
done